DatabaseWorkshop.md

```diff
@@ -0,0 +1,185 @@
+# Adding persistance to ToDoBackend with Swift-Kuery-ORM
+
+<p align="center">
+<img src="https://www.ibm.com/cloud-computing/bluemix/sites/default/files/assets/page/catalog-swift.svg" width="120" alt="Kitura Bird">
+</p>
+
+<p align="center">
+<a href= "http://swift-at-ibm-slack.mybluemix.net/"> 
+    <img src="http://swift-at-ibm-slack.mybluemix.net/badge.svg"  alt="Slack"> 
+</a>
+</p>
+
+## Prerequisite
+This is a follow on tutorial to our [ToDoBackend tutorial](https://github.com/IBM/ToDoBackend). Please complete that before proceeding with this tutorial.
+
+## Installing PostgreSQL
+
+Before we start, ensure you have PostgreSQL installed and running. In Terminal, run:
```

Comment:
Why don't we simply all of this section to the following:

In this tutorial you'll be using the [Swift Kuery PostgreSQL plugin](https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL), so you will need PostgreSQL running on your local machine. You can install and start PostgreSQL as follows:

```
 brew install postgresql
 brew services start postgresql
 ```

Reply:
✅